pyproject.toml

```diff
@@ -36,10 +36,10 @@ scipy = "~1.10.1"
 discretize = "~0.10.0"
 
 ## pip dependencies
-geoh5py = {version = "~0.9.0a1", source = "pypi", allow-prereleases = true}
+geoh5py = {version = "~0.9.0-alpha.1", source = "pypi", allow-prereleases = true}
```

Comment:
did the 0.9.* not work?

Reply:
No, conda-lock always gave me a solver error when doing "run_conda_lock.py". I'm not sure if conda understands the same version specs as poetry. I tried some conda ones like ">=0.9.0,<0.10.0" but it didn't like those either.